packages/ui/src/components/logo-v2/logo-v2.tsx

```diff
@@ -37,24 +37,26 @@ interface LogoFallbackPropsV2 extends BaseLogoPropsV2 {
 
 export type LogoPropsV2 = LogoDefaultPropsV2 | LogoFallbackPropsV2
 
-const LogoV2: FC<LogoPropsV2> = ({ name, size, className, skipSize = false, fallback }) => {
-  const Component = name ? LogoNameMapV2[name] : undefined
-  const sizeClasses = skipSize ? '' : logoVariants({ size })
+const LogoV2 = forwardRef<SVGSVGElement, LogoPropsV2>(
+  ({ name, size, className, skipSize = false, fallback, ...props }, ref) => {
+    const Component = name ? LogoNameMapV2[name] : undefined
+    const sizeClasses = skipSize ? '' : logoVariants({ size })
 
-  if (!Component && fallback) {
-    console.warn(`Logo "${name}" not found, falling back to "${fallback}".`)
-    const FallbackComponent = LogoNameMapV2[fallback]
+    if (!Component && fallback) {
+      console.warn(`Logo "${name}" not found, falling back to "${fallback}".`)
+      const FallbackComponent = LogoNameMapV2[fallback]
 
-    return <FallbackComponent className={cn(sizeClasses, className)} />
-  }
+      return <FallbackComponent className={cn(sizeClasses, className)} ref={ref} {...props} />
+    }
 
-  if (!Component) {
-    console.warn(`Logo "${name}" not found in LogoNameMapV2.`)
-    return null
-  }
+    if (!Component) {
+      console.warn(`Logo "${name}" not found in LogoNameMapV2.`)
+      return null
+    }
 
-  return <Component className={cn(sizeClasses, className)} />
-}
+    return <Component className={cn(sizeClasses, className)} ref={ref} {...props} />
```

Comment:
same here. can we remove `{...props}`? Please ensure current functionality is not getting affected.